surfactant/cmd/plugin.py

```diff
@@ -0,0 +1,15 @@
+import click
+
+from surfactant.plugin.manager import get_plugin_manager
+
+
+@click.command(name="list")
+def display():
```

Comment:
Let's also update the function name from `display` to something more specific like `plugin_list_cmd`.

Also let's add a docstring as a line below it with the `"""` syntax (similar to the "Manage plugins" one added for the plugin command group) so that `surfactant plugin --help` will show a brief description of what the list subcommand does.

Reply:
Thank you. I made the changes.